packages/@aws-cdk-testing/framework-integ/test/aws-stepfunctions/test/integ.map-distributed-iam.ts

```diff
@@ -0,0 +1,59 @@
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+import * as cdk from 'aws-cdk-lib';
+import * as sfn from 'aws-cdk-lib/aws-stepfunctions';
+
+/**
+ * Stack verification steps:
+ *
+ * -- aws stepfunctions describe-state-machine --state-machine-arn <stack-output> has a status of `ACTIVE`
+ */
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'cdk-stepfunctions-map-distributed-stack');
+let index = 0;
+
+function createMap() {
+  index++;
+
+  return new sfn.DistributedMap(stack, `Map ${index}`, {
+    stateName: 'My-Map-State',
+    maxConcurrencyPath: sfn.JsonPath.stringAt('$.maxConcurrency'),
+    itemsPath: sfn.JsonPath.stringAt('$.inputForMap'),
+    itemSelector: {
+      foo: 'foo',
+      bar: sfn.JsonPath.stringAt('$.bar'),
+    },
+  }).itemProcessor(new sfn.Pass(stack, `Pass State ${index}`), {
+    mode: sfn.ProcessorMode.DISTRIBUTED,
+    executionType: sfn.ProcessorType.STANDARD,
+  });
+}
+
+const sm1 = new sfn.StateMachine(stack, 'StateMachine1', {
+  definitionBody: sfn.DefinitionBody.fromChainable(createMap()),
+  timeout: cdk.Duration.seconds(30),
+});
+
+new cdk.CfnOutput(stack, 'StateMachine1ARN', {
```

Comment:
These outputs are great but we should do something with them. Perhaps add assertions that get the machine state of each of these?